webapp/channels/src/packages/mattermost-redux/src/reducers/entities/posts.ts

```diff
@@ -345,68 +377,70 @@ export function handlePosts(state: IDMappedObjects<Post> = {}, action: MMReduxAc
 function handlePostReceived(nextState: any, post: Post, nestedPermalinkLevel?: number) {
     let currentState = nextState;
 
+    const processedPost = post;
```

Comment:
Why are we not using the original variable? I don't see any other usage of the original `post` variable other than on this line.